### PR TITLE
handling the QuickSlow crossword

### PR DESCRIPTION
**Pull request by railsagainstignorance**
Each QuickSlow instance is one of 6 standard templates

### CODE REVIEW

File: crosswordDSL/QuickSlow/example1.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Quick Slow 375 issue 03/03/18 / [...] Please credit ‘Aldhelm’ / 1 of 4
```
Comment (Lily2point0) [inline crosswordDSL/QuickSlow/example1.txt:1]: Was this file meant to be committed?

File: crosswordDSL/QuickSlow/grid1.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+name: QuickSlow grid1 2018/03/01
+author: QuickSlow
```
Comment (Lily2point0) [inline crosswordDSL/QuickSlow/grid1.yaml:2]: Author name doesn't show as per the example text file

File: crosswordDSL/QuickSlow/example1.txt (hunk 1 above)
Comment (railsagainstignorance) [reply crosswordDSL/QuickSlow/example1.txt:1]: yes, as an example.

Comment (railsagainstignorance) [reply crosswordDSL/QuickSlow/example1.txt]: from the future.
hm.

File: crosswordDSL/QuickSlow/grid1.yaml (hunk 2 above)
Comment (railsagainstignorance) [reply crosswordDSL/QuickSlow/grid1.yaml:2]: These are overwritten by the specific crossword. 
When a new crossword comes in, the appropriate grid is identified, and just the coords are copied across into the crossword obj.

**Pull request merged**